script/test

```diff
@@ -71,7 +71,7 @@ for (let packageName in CONFIG.appMetadata.packageDependencies) {
     console.log(`Executing ${packageName} tests`.bold.green)
     const cp = childProcess.spawn(executablePath, testArguments)
     let stderrOutput = ''
-    cp.stderr.on('data', data => stderrOutput += data)
+    cp.stderr.on('data', data => (stderrOutput += data))
```

Comment:
Yes, you're right.  I've been using curly brackets in other places to fix the warning but not sure why I didn't here.